### PR TITLE
Delete vote deletes the vote with all of its belongings #45

### PR DESCRIPTION
**Pull request by valentinbujdoso**
Delete vote deletes the vote with all of its belongings
Fix some behaviour

### CODE REVIEW

File: src/test/java/org/rulez/demokracia/pdengine/VoteDeleteValidationTest.java [hunk 1]
```diff
@@ -43,6 +43,19 @@ public void invalid_adminKey_is_rejected() throws ReportedException {
 	@tested_operation("delete vote")
 	@tested_behaviour("validate inputs")
 	@Test
+	public void a() throws ReportedException {
```
Comment (magwas) [inline src/test/java/org/rulez/demokracia/pdengine/VoteDeleteValidationTest.java]: No meaningful test name.
This contradicts to the behaviour "deletes the vote with all parameters, choices, ballots and votes cast"
https://adadocs.demokracia.rulez.org/PDEngine/edemo/master/index.html#4c75a22b-39e3-493b-b874-505eaf816282

**Review by magwas — CHANGES_REQUESTED** [review]

File: src/test/java/org/rulez/demokracia/pdengine/VoteDeleteValidationTest.java (hunk 1 above)
Comment (valentinbujdoso) [reply src/test/java/org/rulez/demokracia/pdengine/VoteDeleteValidationTest.java]: Fixed

**Review by magwas — APPROVED** [review]
ok

**Comment by magwas** [comment]
testcaseCount=1

**Pull request merged**